backend/utils.ts

```diff
@@ -13,3 +14,20 @@ export async function hashPassword(password: string): Promise<string> {
   const hashedPassword = await hash(password, saltRounds);
   return hashedPassword;
 }
+
+let loopInterval: NodeJS.Timeout | undefined = undefined;
+
+// 24 hours
+const timeoutTime = 24 * 60 * 60 * 1000;
+
+export async function deleteExpiredSessionsLoop() {
+  if (loopInterval === undefined) {
+    // run delete function immediately, otherwise would need to wait 24 hours for the first run
+    await lucia.deleteExpiredSessions();
+    loopInterval = setInterval(() => {
+      lucia.deleteExpiredSessions().catch((e) => {
+        console.error(e);
+      });
+    }, timeoutTime);
+  }
+}
```

Comment:
Laitoin tämän funktion `utils.ts`-tiedostoon, onko ihan ok?

tein myös tarkoituksella globaalin funktion `loopInterval`, koska halutaan tietää onko looppi jo päällä

Laitoin myös niin, että kun looppi laitetaan tulille, ajetaan vanhojen Sessioneiden poisto samalla. Seuraava siitä on 24h päästä

En laittanut loopin sisällä olevaa funktiota `await`, koska ei pitäisi olla tarpeellinen. Jos kuitenkin laittoi `await` ja  funktion `async`:in, heitti ESLint seuraavan virheen [Promise returned in function argument where a void return was expected](https://stackoverflow.com/questions/63488141/promise-returned-in-function-argument-where-a-void-return-was-expected)

Reply:
Joo, oli näköjään jäänyt poistamatta :see_no_evil: